rwengine/src/ai/TrafficDirector.hpp

```diff
@@ -1,12 +1,11 @@
-#pragma once
+#ifndef _TRAFFICDIRECTOR_HPP_
+#define _TRAFFICDIRECTOR_HPP_
```

Comment:
This one is missing the module name

Reply:
Ok, did not look at that. I'll use `_{MODULE}_{FILENAME}_HPP_`